src/main/java/com/cdancy/bitbucket/rest/binders/BindSearchRequestToPayload.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.binders;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.cdancy.bitbucket.rest.domain.search.SearchRequest;
+
+import javax.inject.Singleton;
```

Comment:
:warning: Extra separation in import group before 'javax.inject.Singleton' (#QR41:24#)-CHECKSTYLE_CustomImportOrderCheck